gabbi/tests/test_verbose.yaml

```diff
@@ -0,0 +1,9 @@
+tests:
```

Comment:
I didn't realize when merging this that this file was not actually being used. I'll fix it while dealing with #182